ext/node/polyfills/process.ts

```diff
@@ -691,6 +690,12 @@ if (isWindows) {
 /** https://nodejs.org/api/process.html#process_process */
 const process = new Process();
 
+Object.defineProperties(process, {
```

Comment:
Lazily creating these chops a bit of time off the v8 initialization profile. 

Question: these were previously exported directly in the module. Do we also need to `export = process`?

Old code was:

```ts
export { stderr, stdin, stdout };
```